front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/OpenToParticipation/Settings.tsx

```diff
@@ -1,36 +1,13 @@
 import React from 'react';
 
 import { Box } from '@citizenlab/cl2-component-library';
-import { useNode } from '@craftjs/core';
 
-import InputMultilocWithLocaleSwitcher from 'components/UI/InputMultilocWithLocaleSwitcher';
-
-import { useIntl } from 'utils/cl-intl';
-
-import messages from './messages';
+import TitleMultilocInput from '../_shared/TitleMultilocInput';
 
 const Settings = () => {
-  const { formatMessage } = useIntl();
-
-  const {
-    actions: { setProp },
-    titleMultiloc,
-  } = useNode((node) => ({
-    titleMultiloc: node.data.props.titleMultiloc,
-  }));
-
   return (
     <Box my="20px">
-      <InputMultilocWithLocaleSwitcher
```

Comment:
Converted this to a reusable component, so that I can easily use it in the Followed items widget settings too